mlrun/feature_store/feature_set.py

```diff
@@ -405,7 +405,7 @@ def uri(self):
 
     @property
     def fullname(self) -> str:
-        """full name in the form {project}/{name}[:{tag}]"""
+        """full name in the form [project/]name[&#58;tag]"""
```

Comment:
Seems to me like what's there now is correct(*) but needs escaping, as in:
```suggestion
        """full name in the form ``{project}/{name}[:{tag}]``"""
```
Otherwise it still comes out wrong in the HTML:
```
full name in the form [project/]name[&#58;tag]
```

(*) The notation {...} indicates substitution, while the notation [...] indicates optionality.